python-sdk/src/astro/databases/base.py

```diff
@@ -81,23 +82,36 @@ def run_sql(
         self,
         sql_statement: str | ClauseElement,
         parameters: dict | None = None,
+    ):
+        warnings.warn(
+            "`run_sql` is deprecated and will be removed "
+            "Please use  `run_sql_query` instead.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return self.run_sql_query(sql=sql_statement, parameters=parameters)
+
+    def run_sql_query(
+        self,
+        sql: str | ClauseElement,
+        parameters: dict | None = None,
     ):
         """
         Return the results to running a SQL statement.
 
         Whenever possible, this method should be implemented using Airflow Hooks,
         since this will simplify the integration with Async operators.
 
-        :param sql_statement: Contains SQL query to be run against database
+        :param sql: Contains SQL query to be run against database
         :param parameters: Optional parameters to be used to render the query
         """
         if parameters is None:
             parameters = {}
 
-        if isinstance(sql_statement, str):
-            result = self.connection.execute(sqlalchemy.text(sql_statement), parameters)
+        if isinstance(sql, str):
```

Comment:
```suggestion

        if "sql_statement" in kwargs:
            warnings.warn(
                "`sql_statement` is deprecated and will be removed in future release"
                "Please use  `sql` param instead.",
                DeprecationWarning,
                stacklevel=2,
            )
        sql = sql or kwargs.get("sql_statement")    
        if isinstance(sql, str):
```

Reply:
^^ something like this (untested)

Reply:
Then it won't be a breaking change either